### ISSUE TITLE
There is no notification when scheduled toots fail 

### ISSUE THREAD

**Issue by deutrino** [stom79/mastalab#285]
I've had some scheduled toots fail and the only indication of this was the red text "failed!" in the scheduled toots list. 

I suggest a notification if a scheduled toot failed (either due to an error posting, or because the device was powered off, etc.

**Comment by stom79** [comment]
Yes. This point needs to be improved and I guess it's related to your last issue. Most of the time, it fails due to battery optimization, when Internet connection is turned off.

**Comment by deutrino** [comment]
My device has been powered on whenever I got the failures, but I'm not sure the instance was working properly. But it would be easy for a dead battery to cause scheduled toots to be missed!

Further suggestion, maybe an "edit" and "post now" function in the notification - tap to edit, and a post now button
